layouts/New/layouts.module.css

```
@@ -0,0 +1,62 @@
+.baseLayout {
+  @apply grid
+    h-screen
+    w-screen
+    grid-cols-[1fr]
+    grid-rows-[auto_1fr_auto];
+}
+
+.articleLayout {
+  @apply grid
+    grid-rows-[1fr]
+    overflow-hidden
+    grid-areas-[sidebar_main_metabar,sidebar_footer_metabar]
+    sm:grid-cols-[auto_1fr_auto]
+    xl:grid-cols-[320px_1fr_320px]
+    xs:block
+    xs:overflow-auto;
+
+  > *:nth-child(1) {
+    @apply grid-in-[sidebar];
+  }
+
+  > *:nth-child(2) {
+    @apply flex
+      w-full
+      flex-col
+      items-start
+      gap-8
+      self-stretch
+      overflow-y-auto
+      overflow-x-hidden
+      bg-gradient-subtle
+      px-12
+      py-12
+      grid-in-[main]
+      dark:bg-gradient-subtle-dark
+      xs:!bg-none
```

```suggestion
      xs:bg-none
      xs:dark:bg-none
```

We can style it without important